distribution/docker/build.gradle

```diff
@@ -0,0 +1,23 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
+description = 'Dockerized SAML CKT.'
+
+dependencies {
+    compile project(':distribution:command-line')
+    compile project(':distribution:suites')
+}
+
+// Copy the distribution from the distribution module over to the build module
+task copyDistribuion(type: Copy) {
```

Comment:
✏️ "copyDistribution" missing a "t"
❓ What is the purpose of doing this copy?

Reply:
We need to copy the distribution over to the `docker` so the Dockerfile can have access to it. We were doing the same in maven too.